tape/src/main/java/com/squareup/tape2/FileObjectQueue.java

```diff
@@ -1,17 +1,17 @@
 // Copyright 2012 Square, Inc.
 package com.squareup.tape2;
 
-import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.util.Iterator;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import okio.Buffer;
 
 final class FileObjectQueue<T> extends ObjectQueue<T> {
   /** Backing storage implementation. */
   private final QueueFile queueFile;
   /** Reusable byte output buffer. */
-  private final DirectByteArrayOutputStream bytes = new DirectByteArrayOutputStream();
+  private final Buffer buffer = new Buffer();
```

Comment:
Since Okio pools I don't think keeping this instance and reusing it buys anything.

Reply:
good catch.